app/scripts/background.js

```diff
@@ -273,7 +274,7 @@ function setupController (initState, initLangCode) {
     asStream(controller.store),
     debounce(1000),
     storeTransform(versionifyData),
-    storeTransform(persistData),
```

Comment:
So a transform stream will just stop if it doesn't get read from?

Reply:
it has a `highWatermark` option which defaults to 16